src/components/Input.js

```diff
@@ -3,13 +3,12 @@ import PropTypes from 'prop-types'
 
 const InputStyled = styled.input`
     width: 100%;
-    padding: 0 10px;
-    border-top-left-radius: 5px;
+    position: relative;
+    padding-left: 10px;
+    padding-right: 30%;
+    border-radius: 5px;
     border-bottom-left-radius: 5px;
-    border-top: 1px solid var(--gray);
-    border-bottom: 1px solid var(--gray);
-    border-left: 1px solid var(--gray);
-    border-right: 1px solid white;
+    border: 1px solid var(--gray);
```

Comment:
Do you use e.g. a main color or/ and a secondary color? If that is the case, you could change your color variable names to more general ones. Doing this, it is even more handy to change main color themes.

Reply:
In reason of more than two colors, I decided to name it by the color, than the content is clear.